docs/postgres.md

```diff
@@ -41,3 +41,7 @@ db.postgres.timeout=0
 ```
 Connection timeout is disabled, to account for situations where the database
 might be slow for unexpected reasons.
+
+## Important note about replication
+
+In case a replication architecture is planned, streaming replication should be avoided, as the master does not verify the replica is indeed identical, but it will only forward the edits queue, and let the slave catch up autonomously; synchronous mode, albeit slower, is paramount for `lnd` data integrity across the copies, as it will finalize writes only after the slave confirmed successful replication.
```

Comment:
There are lots of commas in the structure of this sentence.

'should be avoided' leaves some room for doing streaming replication still.

> NEVER use streaming replication. With streaming replication, the replica data may be outdated for brief periods of time. If you'd need to recover from the replica, there is a risk of breaching channels and losing funds!
>
> The only safe way to use replication is using synchronous mode.

Reply:
> There are lots of commas in the structure of this sentence.

Yes, I can be a sucker for correct pauses along sentences, and prefer to err on the side of correct punctuation.

> 'should be avoided' leaves some room for doing streaming replication still.

Wouldn't want to impose my reasoning, I thought that once the user has been educated about the downsides of an alternative, if they still choose it, it's on them in case they pay the consequences; personally, I wouldn't want a software to tell me what I can or can't do, and I would prefer to know what instead I should or shouldn't, and why.
Streaming replication has still a place in a multiple slave config, where one is sync and the other is async, and/or in case multiple failsafe mechanisms are in place, UPS and RAID1 for example; I have both, but now that I know what streaming replication really is (I didn't before) I also want to use synchronous mode exclusively.

@guggero I'm sorry, I mean it. Last time I used the checklists, this time I deliberately ignored them ("eh it's just a doc after all").